projects/plugins/jetpack/extensions/blocks/subscriptions/subscriptions.php

```diff
@@ -720,7 +720,7 @@ function render_for_website( $data, $classes, $styles ) {
 	<div <?php echo wp_kses_data( $data['wrapper_attributes'] ); ?>>
 		<div class="wp-block-jetpack-subscriptions__container<?php echo ! $is_subscribed ? ' is-not-subscriber' : ''; ?>">
 			<?php if ( is_top_subscription() ) : ?>
-				<p id="subscribe-submit"
+				<p id="subscribe-link"
```

Comment:
How come we use ID here instead of class? 🤔 Makes me thinking something is using to scroll to it later on.

Not a super biggie, but ID is also problematic for having multiple blocks on the page since there should always be just one unique ID per page load.

Reply:
Agree, but at the same time, I want to be consistent with the rest of the code.

Not sure why we have IDs there. I prefer to remove them in a separate PR if we want to do it.

Reply:
There are already some styles attached to it in premium themes:
fbhepr%2Sfrnepu%3Sshyy%3Q%2522fhofpevor%2Qfhozvg%2522%26qrsf%3Q%26ersf%3Q%26cngu%3Q%26uvfg%3Q%26glcr%3Q%26keq%3Q%26aa%3Q17%26fv%3Qshyy%26frnepunyy%3Qgehr%26fv%3Qshyy-og

Reply:
Can we keep the submit ID but start using something else for this new case? E.g. `is-subscribed` class?

Reply:
Like the idea, will try that 👍